source/class/qx/tool/compiler/app/ManifestFont.js

```diff
@@ -0,0 +1,171 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2023 Zenesis Limited https://www.zenesis.com
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+     * John Spackman (@johnspackman)
+
+************************************************************************ */
+
+const fs = require("fs");
+const path = require("path");
+
+/**
+ * Represents a font as defined in the Manifest.json's
+ */
+qx.Class.define("qx.tool.compiler.app.ManifestFont", {
+  extend: qx.core.Object,
+
+  construct(name) {
+    super();
+    this.setName(name);
+  },
+
+  properties: {
```

Comment:
No JSDoc description for properties

Reply:
what are you proposing?  That it says:

```
/* these are properties */
properties: {
```

Do we have that kind of comment in every other file??

Reply:
I try to explain. The class `qx.tool.compiler.app.ManifestFont` has `properties` section and every property without JSDoc comment section above it (Properties ). Ofc, I could write the same phrase for each property but does it worth it? I remember at one of my PR I was asked to add jsdoc for properties.